components/nimbus/android/lint-baseline.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="6" by="lint 8.12.1" type="baseline" client="gradle" dependencies="false" name="AGP (8.12.1)" variant="all" version="8.12.1">
+
+    <issue
```

Comment:
Doesn't block this patch, but surely we can get a ticket to fix these trivial lint issues?